fdbserver/TLogServer.actor.cpp

```diff
@@ -1009,11 +1009,9 @@ ACTOR Future<Void> tLogPopCore( TLogData* self, Tag inputTag, Version to, Refere
 	if (self->ignorePopRequest) {
 		TraceEvent(SevDebug, "IgnoringPopRequest").detail("IgnorePopDeadline", self->ignorePopDeadline);
 
-		if (self->toBePopped.find(inputTag) == self->toBePopped.end()
-			|| to > self->toBePopped[inputTag]) {
-			self->toBePopped[inputTag] = to;
-		}
-		// add the pop to the toBePopped map
+		auto& v = self->toBePopped[inputTag];
+		v = std::max(v, to);
```

Comment:
You may need to move this out of here now, since `toBePopped` will now be empty when you call it in `processPopRequests`. The only place where this logic has an effect is the bottom of `tLogPop()`, currently line 1088.